README.md

```diff
@@ -30,7 +30,7 @@ How to install and run Brackets
 
 Installers for the latest stable build for Mac, Windows and Linux (Debian/Ubuntu) can be [downloaded here](http://brackets.io/).
 
-The Linux version has most of the features of the Mac and Windows versions, but
+The Linux version has more features than the Mac and Windows versions, but
```

Comment:
Maybe a better change would be: "The Linux version has almost all the features found in the Mac and Windows versions, but..."